lib/ansible/plugins/strategy/__init__.py

```diff
@@ -313,29 +298,12 @@ def run(self, iterator, play_context, result=0):
                 except KeyError:
                     iterator.get_next_task_for_host(self._inventory.get_host(host))
 
-        # save the failed/unreachable hosts, as the run_handlers()
-        # method will clear that information during its execution
-        failed_hosts = iterator.get_failed_hosts()
-        unreachable_hosts = self._tqm._unreachable_hosts.keys()
-
-        display.debug("running handlers")
-        handler_result = self.run_handlers(iterator, play_context)
-        if isinstance(handler_result, bool) and not handler_result:
-            result |= self._tqm.RUN_ERROR
```

Comment:
I thought it could use a note in the changelog, but I don't see how run_handlers would have returned False. I guess this was unused?

Reply:
Yeah, the return value of `run_handlers` was effectively hardcoded to `RUN_OK`. However I think you still have a point because since we now run handlers through strategies they affect the return value of each strategy which will be different from devel.

Reply:
There seems to be no change in behavior. There is a check below for failed hosts and there the `result` (and subsequently the return code) is changed to be non-zero so I think we're good here.

Reply:
Okay. It *looked* like a change in behavior, but I couldn't find one when tracing the code either. Thanks for checking.